docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -65,60 +66,284 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
 export class AuthConfigModule {}
 ```
 
-## Passing AuthWellKnown
+## Config Values
 
-The third property is `authWellKnown` where you can pass the AuthWellKnownEndpoint if you have it already, otherwise it will be loaded automatically for you with the values from the config you provide.
+### `configId`
 
-```ts
-AuthModule.forRoot({
-      authWellKnown: {
-        /* YOUR AUTHWELLKNOWN VALUES HERE */
-      },
-    }),
-```
+- Type: string
+- Required: false
 
-## Config Values
+To identify a configuration a new parameter called `configId` was introduced. This is being filled from the lib when the user does not set it explicitly in the config. If set, the configured value is taken. It is optional. For a single config this param is not really important. For multiple configs this param is important.

```

Comment:
I would rephrase for clarity, like so:

To identify a configuration a new parameter called `configId` was introduced. If you do not explicitly set this value, the library will generate and assign the value for you. If set, the configured value is used. The value is optional, and for single-configuration projects, `configId` is largely unimportant. For multiple-configuration projects, it is important to set a unique `configId` for each separate configuration.

(Note: If it's required to assign a `configId` for multi-config projects, I would also add that to the above verbiage)